nft/collection/mint_process.go

```diff
@@ -59,8 +59,17 @@ func (ipp *MintItemProcessor) PreProcess(
 		return err
 	} else if !design.Active() {
 		return errors.Errorf("deactivated collection; %q", design.Symbol())
-	} else if !ipp.sender.Equal(design.Creator()) {
-		return errors.Errorf("sender must be collection creator; creator: %q", design.Creator().String())
+	} else if policy, ok := design.Policy().(CollectionPolicy); !ok {
+		return errors.Errorf("policy of design is not collection-policy; %q", design.Symbol())
+	} else if whites := policy.Whites(); !design.Creator().Equal(ipp.sender) {
+		for i := range whites {
+			if whites[i].Equal(ipp.sender) {
+				break
+			}
+			if i == len(whites)-1 {
+				return errors.Errorf("sender is not whitelisted, nor is collection creator; %q", ipp.sender)
```

Comment:
@wyuinche 
I think it's good that the error message informs only one situation. If these two situations are included together, it makes it difficult to find the exact cause when debugging later.

Reply:
@soonkuk It has been changed so that only whitelisted accounts can mint nft.

Reply:
@wyuinche Ok I got it.